static/js/chart/draw.ts

```diff
@@ -1135,6 +1159,57 @@ function drawGroupLineChart(
       })`
     );
   buildInChartLegend(legend, plotParams.legend, legendTextdWidth);
+
+  // Add highlight on hover
+  const chartAreaBoundary = {
+    bottom: height - bottomHeight,
+    left: leftWidth,
+    right: width - MARGIN.right,
+    top: 0,
+  };
+  const highlightColorFn = (place: string, dataGroup: DataGroup) => {
+    return plotParams.lines[place + statsVarsTitle[dataGroup.label]].color;
+  };
+  const getTooltipContent = (highlightedTime: number): string => {
```

Comment:
maybe define this as "function" to be consistent with elsewhere

Reply:
also would be nice to add function comment about what content it generates

Reply:
oh I see, there are two "getTooltipContent". maybe better two make two functions like getTooltipContentforLine() and getTooltipContentforGroupLine() as standalone functions? And would be better if can consolidate them